numpy/random/mtrand.pyx

```diff
@@ -4079,13 +4079,30 @@ cdef class RandomState:
         >>> x.shape
         (3, 3, 2)
 
-        Here we generate a scatter plot of 800 samples from the bivariate
-        normal distribution with mean (0, 0) and covariance matrix
-        [[6, -3], [-3, 3.5]].
+        Here we generate 800 samples from the bivariate normal distribution
```

Comment:
Ideally any changes made to the above are replicated here.

Reply:
Done.